### PR TITLE
[Fix-10890] fix some bugs in the Resource page

### LINKED ISSUE #10890: [Bug] [UI] some bugs in the `Resource` page
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar issues.

### What happened

 some bugs in the `Resource` page
1. click edit a `file` in `UDF Manage - Resource Manage`. modal Text error
image
2. firstly upload one UDF file succeeded and reupload a file again. button loading default
image
3. UDF files need to limit to one. cause it only supports one upload at once
image

### What you expected to happen

make pages show as expected.

### How to reproduce

First, go to the 'Resouce page - UDF Manage'.
Second, try to edit a file.

### Anything else

_No response_

### Version

dev

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by rockfang**
## Purpose of the pull request

close #10890 
fix some bugs in the `Resource` page

## Brief change log
1.  EditModal in `UDF Manage - Resouce Manage`. Fixed the title and label
2.  Limit upload UDF Resouce to 1 and add validate at the proper time.
3.  Fixed the bug about button loading in UploadUDFModal
4.  In the CreateUDFFunctionModal. set a default value for `type`
## Verify this pull request

This pull request is code cleanup without any test coverage.

### CODE REVIEW

**Comment by rockfang** [comment]
Here are some screenshots 
1. Edit UDF dir
edit-udf-dir
2. Edit UDF file
edit-udf-file
3. UDF function type default checked
udf-function-default-check
4. Limit to 1 in UDFUploadModal
udf-upload-limit1-and-validate-remove

File: dolphinscheduler-ui/src/views/resource/file/upload/index.tsx [hunk 1]
```diff
@@ -55,6 +53,8 @@ export default defineComponent({
 
     const removeFile = () => {
       state.uploadForm.name = ''
+      state.uploadForm.file = ''
+      state.uploadFormRef.validate()
```
Comment (devosend) [inline dolphinscheduler-ui/src/views/resource/file/upload/index.tsx]: No verification is required here.

File: dolphinscheduler-ui/src/views/resource/udf/resource/components/upload-modal.tsx [hunk 2]
```diff
@@ -52,12 +52,20 @@ export default defineComponent({
     const customRequest = ({ file }: any) => {
       state.uploadForm.name = file.name
       state.uploadForm.file = file.file
+      state.uploadFormRef.validate()
+    }
+
+    const removeFile = () => {
+      state.uploadForm.name = ''
+      state.uploadForm.file = ''
+      state.uploadFormRef.validate()
```
Comment (devosend) [inline dolphinscheduler-ui/src/views/resource/udf/resource/components/upload-modal.tsx]: No verification is required here.

File: dolphinscheduler-ui/src/views/resource/file/upload/index.tsx (hunk 1 above)
Comment (rockfang) [reply dolphinscheduler-ui/src/views/resource/file/upload/index.tsx]: Agree. I'll add a commit to cancel the verification.

File: dolphinscheduler-ui/src/views/resource/udf/resource/components/upload-modal.tsx (hunk 2 above)
Comment (rockfang) [reply dolphinscheduler-ui/src/views/resource/udf/resource/components/upload-modal.tsx]: Agree. I'll add a commit to cancel the verification.

**Comment by rockfang** [comment]
I add the `reset form` code to solve the problem in this situation:
Firstly, come in `Resouces - File Manage` page
Secondly, click `Upload File` to choose a file. Then cancel the modal without submitting it.
Thirdly, try to `Upload File` again. the form will show the last messages.
image
@devosend

File: dolphinscheduler-ui/src/views/resource/file/upload/index.tsx [hunk 3]
```diff
@@ -38,15 +38,16 @@ export default defineComponent({
     const { handleUploadFile } = useUpload(state)
 
     const hideModal = () => {
+      state.uploadForm.name = ''
```
Comment (devosend) [inline dolphinscheduler-ui/src/views/resource/file/upload/index.tsx]: The `resetForm ` function above can reset the form. line 37

Comment (rockfang) [reply dolphinscheduler-ui/src/views/resource/file/upload/index.tsx]: 😥 got it!

**Review by devosend — APPROVED** [review]
LGTM

**Pull request merged**